src/python/nimbusml/model_selection/cv.py

```diff
@@ -450,13 +450,22 @@ def fit(
         # Need to infer from group_id, bug 284886
         groups = groups or group_id
         if groups is not None:
-            if groups not in cv_aux_info[0]['data_import'][0].inputs[
-                    'CustomSchema']:
-                raise Exception(
-                    'Default stratification column: ' +
-                    str(groups) +
-                    ' cannot be found in the origin data, please specify '
-                    'groups in .fit() function.')
+            if isinstance(X, FileDataStream):
+                if groups not in cv_aux_info[0]['data_import'][0].inputs[
+                        'CustomSchema']:
+                    raise Exception(
+                        'Default stratification column: ' +
+                        str(groups) +
+                        ' cannot be found in the origin data, please specify '
+                        'groups in .fit() function.')
+            elif isinstance(X, DataFrame):
```

Comment:
>elif isinstance(X, DataFrame): [](start = 12, length = 30)

please also add a test that covers this new path.

Reply:
Sure. Let me add that~